grails-app/taglib/org/pih/warehouse/AuthTagLib.groovy

```diff
@@ -43,6 +43,18 @@ class AuthTagLib {
         }
     }
 
+    def isNonInventoryManagedAndCanSubmitRequest = {attrs, body ->
+        def warehouseInstance = Location.get(session.warehouse.id)
+        def authorized = true
+
+        authorized = authorized && !warehouseInstance.supports(ActivityCode.MANAGE_INVENTORY)
+        authorized = authorized && warehouseInstance.supports(ActivityCode.SUBMIT_REQUEST)
+
+        if (!authorized) {
```

Comment:
Add a comment for this line in the code, that `If the location does not have MANAGE_INVENTORY activity code and have SUBMIT_REQUEST activity code, then return empty body`